<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartitionView</class>
 <widget class="QDialog" name="PartitionView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>select a partition to open</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetPartitionView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>partition list</string>
    </property>
   </column>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
